### DISCUSSION TITLE
Newbie Download Question

### DISCUSSION THREAD

**Discussion by WabashFan** [community/community#34734] [New to GitHub] [answered]
I am a newbie to GitHub & sort of gotten over whelmed  in what I am trying to accomplish.
I do have a GitHub account & have signed in.
I am trying to down load the Raspberry Pi "Code the Classics1" python game programs.
The book "Download" link redirects me to ( https://github.com/Wireframe-Magazine/Code-the-Classics.)
In GitHub I can see the 5 programs listed, I can see the python code, & I can copy the raw data & paste that into the IDE.
I can open the sub files & see the data under them.
I also see the <program name> master folder which I'm guessing is what I really need to down load.
But it will not run because there are subfolders under that did not get copied which are needed when I run the python source.
When I clicked the Raw data button, that means just that it says.
So the question is - how do I get the additional sub folders copied that are /images, /music, & /sounds.
When I "quickly" compare the listing in the book to the listing I pasted into the IDE, I didn't find any differences, i.e. I have the same  number of lines of code.
It will take a line by line look through the listing to see if there are lines commented out in the listing vs the source from GitHub, etc.

I found a similar question asked & answered were I saw "file clone" mentioned, which I saw someplace but I have no clue where I saw that option.  I remember clicking it but nothing happened.  There was HTML. SSH, & some other wording in the message box with HTML high lighted.

Rather than spend time getting further in over my head or screwing something up, tell me where I can read the documentation & get this accomplished.

Hope I have explained this well enough.
Thanks
Dennis Munro

**Comment by WabashFan** [comment, marked as answer]
I found the answer in the back of the book I was downloading from.
It is the git  clone option that I needed.
I know I clicked that but I did not wait long enough I'm guessing because I now have all of the 5 games on my Raspberry Pi on the desktop.

<SNIP>
On the game’s GitHub page, copy the web address that comes up when ‘Clone or download’ is clicked, and then either use git via
the command prompt/terminal to download the project – enter git clone followed by the URL found by clicking on ‘Clone or download’ on the game’s GitHub page; or download it via the ‘Check out from Version Control’ option in PyCharm, if you intend to use that IDE.
<SNIP>

I will have to move them to the correct folder & go from there.

Thanks for your attention
Dennis Munro

**Reply by queenofcorgis** [reply]
Nice work @WabashFan - way to find the answer yourself - how does this self-solve feel?

**Reply by 3kh0** [reply]
Great job!

**Comment by WabashFan** [comment]
Wabash Fan reply's,

One thing I did not mention in my profile was in my business career, I
spent 35+ years working as an IBM Mid-range Business Computer programmer.
Behind my desk were 12 linear feet of manuals & the answer was in them -
someplace.  Back them Google search did not exist so you could & would
spend untold hours finding the answer to your question.  I this case I went
back to the original manual, looked at the table of contents, went to a
promising section, & the answer I needed was there.

Being a noobie, I was afraid of doing something stupid so I asked the
question.

Thank You
Dennis Munro
Berlin, WI

**Comment by Codingcfiu** [comment]
Hi there i am New too
And i never Known anything about programming but i would glad If you could help me to develope my New networking setting up a New coding for comunication two ways hacker between a person that been hacked by the hacker . 

Ours goverment system tech is not releven when we have to comunicated like a robots to not againts the law . Instead of that i glad If you could help me to learn about how to create New cable to study in a private comunication with my trustie hacker in a one hardware system on by my own. 

You know what i mean? 😭😭this is to drastic for us to develope this kind New thing and risking our self future and business with personal finance with out located a right place to our Legacy . 

IT not big mattered but im just need new version to my personal manager be bold and get new colabration with new young generation like you to assist my teammate and i learning more about brandnew hacker developer.

**Comment by telkno882** [comment]
And say your valve power channels and take the power and power shell for the best of your everywhere I did that 15 years ago you can't download nothing all y'all do is send people round around the circles and get their information y'all will never accomplish nothing I hate ghetto **** y'all

**Comment by telkno882** [comment]
So most of the conservation a lot about going over the legal assistance racing how we quit trying to steal people Information man and given what they came on this app to get
